unison-runtime/package.yaml

```diff
@@ -27,11 +27,6 @@ when:
     cpp-options: -DARRAY_CHECK
   - condition: flag(stackchecks)
     cpp-options: -DSTACK_CHECK
-  - condition: flag(optchecks)
-    ghc-options: -O2
-    cpp-options: -DOPT_CHECK
-    dependencies:
-      - inspection-testing
```

Comment:
This had to be moved into the `library:` section, because it introduced an unused dependency in `runtime-tests` otherwise.